scale/storage/brokers/nfs_broker.py

```diff
@@ -5,148 +5,95 @@
 import os
 import shutil
 
-from storage.brokers.broker import OldBroker
-from storage.nfs import nfs_umount, nfs_mount
```

Comment:
Yes, it is still used by ingest and strike. Once the ingest system has been refactored similar to this broker refactor, the nfs module can go away.